rskj-core/src/main/java/org/ethereum/core/TransactionExecutor.java

```diff
@@ -334,12 +310,8 @@ private void execError(String err) {
         executionError = err;
     }
 
-    public void go() {
-        if (!readyToExecute) {
-            return;
-        }
+    private void go() {
 
-        // TODO: transaction call for pre-compiled  contracts
         if (vm == null) {
```

Comment:
Can `vm` ever be `null` now?

Reply:
It's possible if the tx doesn't call any contract